bin/setup

```diff
@@ -41,7 +41,7 @@ class Runner
 
     path.write <<~BODY
       APP_HOST=0.0.0.0
-      APP_URL=http://localhost:4567
+      APP_URL=http://localhost:2300
```

Comment:
:thought_balloon: Yeah, we really need to get rid of these. They are too cumbersome to deal with. That said, specifying the port make as a lot of sense.

In practice, I end up tweaking them to use my local wired IP address because my device won't connect without so it's kind of a wash in practice because you always need to tweak the generated defaults for your local environment.